pyews/core.py

```diff
@@ -44,15 +44,32 @@ def invoke(self, soap_body):
                 auth=(self.userconfiguration.credentials.email_address, self.userconfiguration.credentials.password),
                 verify=True
             )
+
+            __LOGGER__.debug('Response HTTP status code: %s', response.status_code)
+            __LOGGER__.debug('Response text: %s', response.text)
+
             parsed_response = BeautifulSoup(response.content, 'xml')
-            try:
-                if parsed_response.find('ResponseCode').string == 'ErrorAccessDenied':
-                    __LOGGER__.info('{}'.format(parsed_response.find('MessageText').string))
-                if parsed_response.find('ResponseCode').string == 'NoError':
-                    return parsed_response
-            except:
-                if parsed_response.find('ErrorCode').string == 'NoError':
-                    return parsed_response
```

Comment:
I've been gazing to this code for a while but couldn't clearly understood the point author intended to put here. 
Should we check both "ResponseCode" and "ErrorCode"? And what the value they may be? They both can be "NoError" and it means request succeeded and method return content as well? In other case method should return `None`? That what i suggested so far.